build/Rulesets/Roslyn_BuildRules.ruleset

```diff
@@ -21,13 +21,16 @@
     <Rule Id="CA1055" Action="None" />
     <Rule Id="CA1056" Action="None" />
     <Rule Id="CA1061" Action="None" /> <!-- "do not hide base class methods": currently violations in the compiler -->
+    <Rule Id="CA1062" Action="None" />
```

Comment:
@mavasani Let me know if I should open follow-up issues for the disabled rules I added. Or maybe some comment. Thanks

Reply:
Yes, please do so and assign it to me. A single issue tracking enabling of all DFA based rules will suffice.